modules/manage/partials/tiered-storage.adoc

```diff
@@ -1084,6 +1082,22 @@ Remote write uses a proportional derivative (PD) controller to minimize the back
 
 The upload backlog controller prevents Redpanda from running out of disk space. If `remote.write` is set to `true`, Redpanda cannot evict log segments that have not been uploaded to object storage. If the remote write process cannot keep up with the amount of data that needs to be uploaded to object storage, the upload backlog controller increases priority for the upload process. The upload backlog controller measures the size of the upload periodically and tunes the priority of the remote write process.
 
+=== Data archiving
+
+If you only enable remote write on a topic, you have a simple backup to cloud storage that you can use for recovery. In the event of a data center failure, data corruption, or cluster migration, you can recover your archived data from the cloud back to your cluster.
+
+==== Configure data archiving
+
+Data archiving requires a <<set-up-tiered-storage,Tiered Storage configuration>>.
+
+To recover a topic from cloud storage, use xref:manage:topic-recovery.adoc[single topic recovery].
```

Comment:
We agreed a while ago to standardize on 'object storage' instead of 'cloud storage'.

https://github.com/redpanda-data/documentation-private/issues/1317